cli/popper/commands/cmd_version.py

```diff
@@ -1,11 +1,11 @@
 import click
 import popper.utils as pu
-
-from popper.cli import pass_context
+from .. import __version__ as popper_version
+from ..cli import pass_context
 
 
 @click.command('version', short_help='Show version of Popper and exit.')
 @pass_context
 def cli(ctx):
     """Displays version of Popper and exit."""
-    pu.info('Popper 0.6-dev.')
+    pu.info(popper_version)
```

Comment:
Minor thing: info message should be `pu.info('popper version ' + popper_version)`, this is typically what other CLI tools do